website/public/locales/en/leaderboard.json

```diff
@@ -32,5 +32,6 @@
   "week": "Week",
   "weekly": "Weekly",
   "your_account": "Your account",
-  "your_stats": "Your statistics"
+  "your_stats": "Your statistics",
+  "every_languages": "Every Languages"
```

Comment:
You dont need to translate this. It's admin only feature

Reply:
I think that it could be reused in other board (#2084)